README.md

```diff
@@ -270,25 +270,21 @@ Chainlink Functions can be used with Chainlink Automation in order to automatica
 
 1. Create & fund a new Functions billing subscription by running:<br>`npx hardhat functions-sub-create --network network_name_here --amount LINK_funding_amount_here`<br>**Note**: Ensure your wallet has a sufficient LINK balance before running this command.<br><br>
 
-2. Deploy the _AutomationFunctionsConsumer_ client contract by running:<br>`npx hardhat functions-deploy-auto-client --network network_name_here --subid subscription_id_number_here --interval time_between_requests_here --verify true`<br>**Note**: Make sure `<blockexplorer>_API_KEY` environment variable is se - this CLI command verifies the contract using this API KEY. API keys for these services are freely available to anyone who creates an EtherScan, PolygonScan or SnowTrace account.<br><br>
+2. Deploy the _AutomationFunctionsConsumer_ client contract by running:<br>`npx hardhat functions-deploy-auto-client --network network_name_here --subid subscription_id_number_here --interval time_between_requests_here --verify true`<br>**Note**: Make sure `<blockexplorer>_API_KEY` environment variable is set because this CLI command also verifies the client contract (if --verify is true) using this API KEY. API keys for block explorers (eg EtherScan, PolygonScan or SnowTrace) are freely available to anyone who creates an account.<br><br>
 
 3. Encode the request parameters into CBOR and store it on chain with `npx hardhat functions-set-auto-request --network network_name_here  --subid subscription_id_number_here --interval time_between_requests_here --contract 0x_contract_address`.  You can now manually check that your on-chain requests work if you try the manual debugging step #4 or you can go ahead and run Chainlink Automations on your contract by following the next step.  
 
-> ⚠️⚠️⚠️ Keep in mind that this task sets DON-Hosted secrets and expires those secrets after 10 minutes.  If you see error bytes returned to your client which decode to secrets not being found, you should run this functions-set-auto-request command or the functions-upload-secrets-don to refresh the DON-hosted secrets for that slotId.
+> ⚠️Keep in mind that this task sets DON-Hosted secrets and expires those secrets after 10 minutes.  If you see error bytes returned to your client which decode to secrets not being found, you should run this functions-set-auto-request command or the functions-upload-secrets-don to refresh the DON-hosted secrets for that slotId.
```

Comment:
Update this with mention of the TTL CLI arg.

Also missing a space before keep.